moped-editor/src/views/projects/projectView/ProjectComponents/mapUtils.js

```diff
@@ -104,62 +108,73 @@ export const ProjectComponentsSourcesAndLayers = ({
             ...mapStyles["ctn-lines"].layerProps,
             layout: {
               ...mapStyles["ctn-lines"].layerProps.layout,
-              visibility:
-                isEditingComponent && linkMode === "lines" ? "visible" : "none",
+              visibility: isEditingLines ? "visible" : "none",
             },
           }}
         />
       </Source>
 
       <Source
```

Comment:
The diff is a little deceptive here - I switched the order of two sources and layers (`project-lines` and `ctn-points`) so that points appear above lines